telemetry/telescope_ec2_age/instance_uptime.py

```diff
@@ -46,24 +61,53 @@ def describe_instances(instance_list):
         return None
 
 
+def describe_try(instance):
```

Comment:
Maybe not necessary to include the word `try` in this method? All methods should have some sort of exception handling so it's kind of implicit.

Reply:
Rename to `describe_instance` or `describe_single_instance` to make it explicit that this a fallback method for describing a single instance.

Reply:
Also adding a docblock explaining why we use this method as a fallback would be useful.